src/net.cpp

```diff
@@ -129,4 +135,14 @@ std::vector<net_socket> net::get_net_sockets(const std::string& file) const
     return output;
 }
 
+std::vector<net_device> net::get_net_devices(const std::string& file) const
+{
+    auto path = _net_root + file;
+
+    static const size_t HEADER_LINES = 2;
+    std::vector<net_device> output;
+    parsers::parse_lines(path, std::back_inserter(output), parsers::parse_net_device_line, HEADER_LINES);
```

Comment:
[style] Can you please wrap the line before `parsers::parse...` just like other methods in the same file?